### ISSUE TITLE
cy.contains(selector, content) incorrectly parses arguments when content regexp

### ISSUE THREAD

**Issue by dwelle** [cypress-io/cypress#801]
- Cypress Version: 1.0.2
- Browser Version: CH 61

### Is this a Feature or Bug?

bug

### Current behavior:

`cy.contains( selector, content )` regards `selector` as `content`, when `content` is regexp.

### Desired behavior:

Should query `selector` and find `content` within it.

### How to reproduce/Test code:

```
<div class="someClass">
    <div>Confirm</div>
</div>
```

```
cy.contains( ".someClass", /confirm/i );
```

![image](https://user-images.githubusercontent.com/5153846/31948536-79131f08-b8d7-11e7-9e5a-a92699ec5a5b.png)

Works as intended when `content` is supplied as string, and not regex:

```
cy.contains( ".someClass", "Confirm" );
```

**Comment by brian-mann** [comment]
Good find. Should be a 2 liner + a single test to fix

**Comment by jennifer-shehane** [comment]
`cy.contains()` code can be found here: https://github.com/cypress-io/cypress/blob/develop/packages/driver/src/cy/commands/querying.coffee#L278

**Comment by brian-mann** [comment]
Fixed by https://github.com/cypress-io/cypress/pull/848.

**Comment by brian-mann** [comment]
Released in [`1.1.0`](https://docs.cypress.io/guides/references/changelog.html#1-1-0).
